src/utils/fetch.js

```diff
@@ -1,13 +1,24 @@
 import axios from 'axios'
 
+const newsAPI = axios.create({
+    baseURL: 'https://be-nc-news-fz91.onrender.com/api/'
+})
+
 export const getArticles = async() => {
-    const response = await axios.get(`https://be-nc-news-fz91.onrender.com/api/articles/`)
+    const response = await newsAPI.get(`/articles`)
     return response
 }
 
 export const getArticleById = async(id) => {
-    const response = await axios.get(`https://be-nc-news-fz91.onrender.com/api/articles/${id}`)
+    const response = await newsAPI.get(`/articles/${id}`)
+    return response
+}
+
+export const getCommentByArticle = async(id) => {
+    const response = await newsAPI.get(`/articles/${id}/comments`)
```

Comment:
Nice that you have created a new function for getting the comments for each article 👍